### PR TITLE
Refactor CustomScrubber to fix support of data-attr

### PR DESCRIPTION
**Pull request by bricesanchez**

### CODE REVIEW

File: pages/app/presenters/refinery/pages/section_presenter.rb [hunk 1]
```diff
@@ -91,13 +91,16 @@ def initialize(tags, attributes)
...
+      end
 
-          super
-        end
+      def skip_node?(node)
+        node.text?
+      end
+
+      def scrub_attribute?(name)
+        !attributes.include?(name) && !(name =~ /\Adata-[\w-]+\z/).present?
```
Comment (parndt) [inline pages/app/presenters/refinery/pages/section_presenter.rb]: We could use exclude? here

Comment (parndt) [inline pages/app/presenters/refinery/pages/section_presenter.rb]: Could also use 
&& name !~ /\Adata-[\w-]+\z/

Comment (bricesanchez) [inline pages/app/presenters/refinery/pages/section_presenter.rb]: true

**Comment by kreker92** [comment]
It has success. Thanks!

**Pull request merged**